src/submodule.c

```diff
@@ -1916,31 +1921,32 @@ static int submodule_load_from_wd_lite(git_submodule *sm)
 
 /**
  * Returns a snapshot of $WORK_TREE/.gitmodules.
```

Comment:
I'd like to see a short comment about `GIT_ENOTFOUND` here such that people know to expect it in case there are no submodules.